huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go

```diff
@@ -0,0 +1,259 @@
+	region := cfg.GetRegion(d)
+
+	var mErr *multierror.Error
+
+	// getPolicy: Query Organizations policy
+	var (
+		getPolicyHttpUrl = "v1/organizations/policies/{policy_id}"
+		getPolicyProduct = "organizations"
+	)
+	getPolicyClient, err := cfg.NewServiceClient(getPolicyProduct, region)
+	if err != nil {
+		return diag.Errorf("error creating Organizations client: %s", err)
+	}
+
+	getPolicyPath := getPolicyClient.Endpoint + getPolicyHttpUrl
+	getPolicyPath = strings.ReplaceAll(getPolicyPath, "{policy_id}", d.Id())
+
+	getPolicyOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	getPolicyResp, err := getPolicyClient.Request("GET", getPolicyPath, &getPolicyOpt)
+
+	if err != nil {
+		return common.CheckDeletedDiag(d, err, "error retrieving Organizations policy")
+	}
+
+	getPolicyRespBody, err := utils.FlattenResponse(getPolicyResp)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	mErr = multierror.Append(
+		mErr,
+		d.Set("name", utils.PathSearch("policy.policy_summary.name", getPolicyRespBody, nil)),
+		d.Set("content", utils.PathSearch("policy.content", getPolicyRespBody, nil)),
+		d.Set("type", utils.PathSearch("policy.policy_summary.type", getPolicyRespBody, nil)),
+		d.Set("description", utils.PathSearch("policy.policy_summary.description", getPolicyRespBody, nil)),
+		d.Set("urn", utils.PathSearch("policy.policy_summary.urn", getPolicyRespBody, nil)),
+	)
+
+	tagMap, err := getTags(getPolicyClient, policiesType, d.Id())
+	if err != nil {
+		log.Printf("[WARN] error fetching tags of Organizations policy (%s): %s", d.Id(), err)
+	} else {
+		mErr = multierror.Append(mErr, d.Set("tags", tagMap))
+	}
+
+	return diag.FromErr(mErr.ErrorOrNil())
+}
+
+func resourcePolicyUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	updatePolicyChanges := []string{
+		"name",
+		"content",
+		"description",
+	}
+
+	// updatePolicy: update Organizations policy
+	var (
+		updatePolicyHttpUrl = "v1/organizations/policies/{policy_id}"
+		updatePolicyProduct = "organizations"
+	)
+	updatePolicyClient, err := cfg.NewServiceClient(updatePolicyProduct, region)
+	if err != nil {
+		return diag.Errorf("error creating Organizations client: %s", err)
+	}
+
+	if d.HasChanges(updatePolicyChanges...) {
+		updatePolicyPath := updatePolicyClient.Endpoint + updatePolicyHttpUrl
+		updatePolicyPath = strings.ReplaceAll(updatePolicyPath, "{policy_id}", d.Id())
+
+		updatePolicyOpt := golangsdk.RequestOpts{
+			KeepResponseBody: true,
+		}
+
+		updatePolicyOpt.JSONBody = utils.RemoveNil(buildUpdatePolicyBodyParams(d))
+		_, err = updatePolicyClient.Request("PATCH", updatePolicyPath, &updatePolicyOpt)
+		if err != nil {
+			return diag.Errorf("error updating Organizations policy: %s", err)
+		}
+	}
+
+	if d.HasChange("tags") {
+		err = updateTags(d, updatePolicyClient, policiesType, d.Id(), "tags")
+		if err != nil {
+			return diag.FromErr(err)
```

Comment:
looks the error message is fixed to organizations tags, please check it

Reply:
done